lib/uffizzi/cli/preview.rb

```diff
@@ -229,5 +235,23 @@ def handle_preview_interruption(deployment_id, hostname, project_slug)
 
       raise Uffizzi::Error.new("The preview creation was interrupted. #{preview_deletion_message}")
     end
+
+    def display_deployment_data(deployment, success)
```

Comment:
may be methods `display_deployment_data` and `build_deployment_data` should be moved to PreviewService?

Reply:
I would not like to move output logic there. I think a cli command is a good place for that